lint.js

```
@@ -1,3 +1,19 @@
+    for (var i = 0; i < array.length; i++) { // 使用 var 声明循环变量
+        if (result.indexOf(array[i]) === -1) { // 缺少分号
+            result.push(array[i]);
+        }
+    }
+    return result;
+}
+
+// 使用函数时缺少分号
+var numbers = [1, 2, 2, 3, 4, 4, 5]
+var uniqueNumbers = unique(numbers)
+console.log(uniqueNumbers);
```

Biome (1.9.4)

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole module. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'const' instead.


(lint/style/noVar)